src/main/app/README.md

```diff
@@ -14,17 +14,17 @@ Run
 Run
 
     cd src/main/app
-    yarn run start
+    npm run start
     
 then go to the app and provide the correct host
```

Comment:
This comment is not correct anymore ;)

Reply:
yeah right didnt see that :D i'll do a full readme rework tomorrow